### PR TITLE
Auto PR Labeler

### PR DESCRIPTION
**Pull request by tnorling**
Github Workflow for labeling our PRs based on which files they touch. Could be interesting to try out instead of marking it in the title. 

Labels for:
- Node
- Angular
- Angularjs
- msal-browser
- msal-common
- msal@1.x
- samples

Feel free to share thoughts on this or suggest additional labels

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+node:
+  - lib/msal-node/**/*
+angular:
+  - lib/msal-angular/**/*
```
Comment (jasonnutter) [inline .github/labeler.yml:4]: Can you add AngularJS?

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+node:
+  - lib/msal-node/**/*
+angular:
```
Comment (jasonnutter) [inline .github/labeler.yml:3]: There is already an Angular label, is that okay?

Comment (tnorling) [reply .github/labeler.yml:3]: Yes. All of these already exist. If it doesn't work it should get logged in the Actions tab

File: .github/labeler.yml (hunk 1 above)
Comment (tnorling) [reply .github/labeler.yml:4]: Done

File: .github/labeler.yml (hunk 2 above)
Comment (jasonnutter) [reply .github/labeler.yml:3]: Ah right.

File: .github/workflows/label.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/labeler@v2
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}
```
Comment (sameerag) [inline .github/workflows/label.yml:19]: Do we also need individuals to get assigned an issue in this same PR? Or are we scraping that?

Comment (sameerag) [reply .github/workflows/label.yml:19]: Never mind! I see a new PR by @jasonnutter on that.

**Pull request merged**